examples/parsons_civis_twilio_example.py

```diff
@@ -1,9 +1,9 @@
-# This script is an example of using the parsons library to import data from
-# Twilio using the parsons Twilio connector, and then upload the resulting
-# table to Platform using the parsons Civis connector.
+# This script is an example of using the Parsons library to import data from
```

Comment:
sorry, one more nit...can we link to the parsons repo?

Reply:
Done